features/bootstrap/FeatureContext.php

```diff
@@ -421,6 +436,27 @@ private function checkXmlFileContents($path, PyStringNode $text)
         Assert::assertEquals(trim($dom->saveXML(null, LIBXML_NOEMPTYTAG)), $fileContent);
     }
 
+    private function checkJSONFileContents($path, PyStringNode $text)
+    {
+        Assert::assertFileExists($path);
+
+        $fileContent = trim(file_get_contents($path));
+
+        $fileContent = preg_replace('/"time": "\d\.\d{3}"/U', '"time": "-IGNORE-VALUE-"', $fileContent);
+
+        $directorySeparator = DIRECTORY_SEPARATOR;
+        if (DIRECTORY_SEPARATOR === '\\') {
+            $directorySeparator .= DIRECTORY_SEPARATOR;
+        }
+        $text = str_replace('-DIRECTORY-SEPARATOR-', $directorySeparator, $text);
+
+        $data = json_decode((string) $fileContent, true);
```

Comment:
Possibly add the JSON_THROW_ON_ERROR here for clearer failure output if the json itself is broken somehow?